sklearn/feature_selection/_univariate_selection.py

```diff
@@ -85,7 +85,7 @@ def f_oneway(*args):
     ----------
     .. [1] Lowry, Richard.  "Concepts and Applications of Inferential
            Statistics". Chapter 14.
-           http://faculty.vassar.edu/lowry/ch14pt1.html
+           http://facultysites.vassar.edu/lowry/PDF/c14p1.pdf
```

Comment:
maybe better to link to the book's website instead? http://vassarstats.net/textbook/

Reply:
Sounds good to me, I'll change that.